tools/abel/README.md

```diff
@@ -69,33 +68,45 @@ dependencies are met:
   pip_install.bat
   ```
 
+5. Navigate to the [ABEL directory](./)
+
+```
+cd abel
+```
+
 ### Obtain a spreadsheet token for your google sheets account
 
-1. Contact your IoT Technical Program Manager and ask to be added to a GCP service account.
+1. [Initialize gcloud CLI](https://cloud.google.com/sdk/docs/initializing)
+  1. Run `gcloud init`
+  2. Choose option 1: `re-initialize default configuration`
+  3. Choose option 2: Enter your Google credentials
+  4. Choose option 1: Enter your GCP project id
+
+![gcloud init screenshot](../../ontology/docs/figures/tools/gcloud_init.png?raw=true)
 
-2. Create a token request file, `token_request.json` with contents:
+More info can be found in [gcloud docs on authorizing with the gcloud CLI](https://cloud.google.com/sdk/docs/authorizing)
+
+2. Contact your IoT Technical Program Manager and ask to be added to a GCP service account.
+
+3. Create a token request file, `token_request.json` with contents:
 
     ```
-    {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+    {"scope": ["https://www.googleapis.com/auth/spreadsheets"],  "lifetime": "3600s"}
     ```
 
-3. Run the below curl command to obtain a token modifying the command with your
+4. Run the below curl command to obtain a token modifying the command with your
```

Comment:
this sentence is a bit weird? rephrase?

Reply:
Done.

Reply:
` Run the below curl command to obtain a token. You need to modify the command with your`